aiohttp/client_exceptions.py

```diff
@@ -152,6 +152,27 @@ class ClientProxyConnectionError(ClientConnectorError):
     """
 
 
+class UnixClientConnectorError(ClientConnectorError):
+    """Unix connector error.
+
+    Raised in :class:`aiohttp.connector.UnixConnector` if
+        connection to unix socket can not be established.
+    """
+    def __init__(self, path: str, connection_key: ConnectionKey,
+                 os_error: OSError) -> None:
+        self._path = path
+        super().__init__(connection_key, os_error)
+
+    @property
+    def path(self) -> str:
```

Comment:
I think I'd prefer this to be a `pathlib.Path` instance that's only stringified when used.

Reply:
UnixConnector historically works with `str` and has `.path` string property.
I think we should keep the type for consistency.